src/test/java/create_task_use_case/CreateTaskControllerTest.java

```diff
@@ -19,11 +19,11 @@ public void testCreateTask(){
         );
         CreateTaskController controller = new CreateTaskController(interactor);
         CreateTaskOutputData outputData = controller.createTask(
-                "Sunday",  "Sample Task"
+                "Sunday",  "Sampled Task"
         );
-        assertEquals("Sample Task", outputData.getTitle());
+        assertEquals("Sampled Task", outputData.getTitle());
         assertEquals(0, outputData.getDayIndex());
         assertTrue(outputData.isSuccessfullyCreated());
-        assertTrue(DATA_ACCESS.existsByTitle("Sample Task", 0));
+        assertTrue(DATA_ACCESS.existsByTitle("Sampled Task", 0));
```

Comment:
This test case didn't fail when I tried to run it in main, but it would fail in its previous state due to the current implementation of DataAccess days as a static variable and the order of your test cases running, which only exposed the issue in the task completion use case.